src/openrct2-ui/windows/RideList.cpp

```diff
@@ -696,6 +705,30 @@ class RideListWindow final : public Window
                                                                          : STR_GUESTS_FAVOURITE_PLURAL_LABEL;
                     }
                     break;
+                case INFORMATION_TYPE_EXCITEMENT:
+                    formatSecondary = STR_RAITING_NOT_YET_KNOWN_LABEL;
+                    if (ridePtr->excitement != RIDE_RATING_UNDEFINED)
+                    {
+                        formatSecondary = STR_EXCITEMENT_LABEL;
+                        ft.Add<uint16_t>(ridePtr->excitement);
+                    }
+                    break;
+                case INFORMATION_TYPE_INTENSITY:
+                    formatSecondary = STR_RAITING_NOT_YET_KNOWN_LABEL;
+                    if (ridePtr->excitement != RIDE_RATING_UNDEFINED)
```

Comment:
```suggestion
                    if (ridePtr->intensity != RIDE_RATING_UNDEFINED)
```

Reply:
For some reason, this returns the actual intensity rating whilst the ride's rating is still undefined. Because of that, I tried to follow the same paradigm as in Ride.ccp lines 5623 to 5645 to get it to work. There, for intensity and nausea, we always check against excitement instead of intensity and nausea.

Reply:
Could you change it to `ride_has_ratings` then its abundantly clear what it means. (We should also clean up the ride window)

Reply:
Thank you, that makes sense. I've just done it and also cleaned the Ride window as you suggested to follow the same pattern. Let me know how it looks to you.